integrations/linear/src/definitions/actions.ts

```diff
@@ -109,9 +109,30 @@ const updateIssue = {
   },
 }
 
+const createIssue = {
+  title: 'Create Issue',
+  input: {
+    schema: z.object({
+      title: z.string().min(2).max(1000),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏ remove all limits since our api already define them